app/services/AuditService.scala

```diff
@@ -126,6 +128,43 @@ class AuditService @Inject() (auditConnector: AuditConnector)(implicit ec: Execu
   )(implicit r: AuthenticatedJourneyRequest[_], hc: HeaderCarrier): Unit =
     audit(toDdinProgressAuditDetail(journey, hasChosenToContinue))
 
+  def auditCanUserPayInSixMonths(journey: Journey, canPay: CanPayWithinSixMonths)(implicit hc: HeaderCarrier): Unit = {
+    audit(toCanUserPayInSixMonths(journey, canPay))
+  }
+
+  private def toCanUserPayInSixMonths(journey: Journey, canPay: CanPayWithinSixMonths): CanUserPayInSixMonthsAuditDetail = {
+    CanUserPayInSixMonthsAuditDetail(
+      regime             = journey.taxRegime.entryName,
+      taxIdentifier      = taxIdentifierToAudit(journey),
+      pegaCaseId         = journey.pegaCaseId,
+      correlationId      = journey.correlationId,
```

Comment:
do we need another field for the correlation id we sent to PEGA in the start case API request? In another audit event for the get case response we added in `pegaCorrelationId` which in the ticket was originally just `correlationId` but we changed it cos it was confusing with the existing `correlationId` you've made use of here

Reply:
🤷🏻  Do you recon that this correlationId refers to the pegaCorrelationId, or that they missed including in?